### ISSUE TITLE
 прямые ссылки на видео?

### ISSUE THREAD

**Issue by akula222** [jumper423/yii2-vk#7]
А можно получить прямые ссылки на видео?  по url видео

**Comment by jumper423** [comment]
@akula22 да, можно. Как я понимаю у вас есть ссылка на проигрыватель с видео.
Она имеет вид https://vk.com/video?z=video-147415323_456239017 её необходимо преобразовать в:
videos = 147415323_456239017

Документация:
https://vk.com/dev/video.get
Если внимательно прочитать можно найти следующее
_Если в Вашем приложении используется прямая авторизация, возвращается дополнительное поле files, содержащее ссылку на файл с видео (если ролик размещен на сервере ВКонтакте) или ссылку на внешний ресурс (если ролик встроен с какого-либо видеохостинга)._
